streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStreamThread.java

```diff
@@ -160,7 +160,8 @@ private StateConsumer initialize() {
                                                                           stateMgr,
                                                                           streamsMetrics,
                                                                           cache),
-                                                                  stateMgr),
+                                                                  stateMgr
```

Comment:
Is this intentional? Or did you just want to align line 163 with previous lines?